run_ci.sh

```diff
@@ -63,7 +63,7 @@ mount | grep root-
 # The CI host has a "/" filesystem that reset for each job, the only partition
 # that persist is /opt (and /boot), we can use this to store data between jobs
 FILECACHEDIR=/opt/data/filecache
-FILESTOCACHE="/opt/dev-scripts/ironic/html/images/ironic-python-agent.initramfs /opt/dev-scripts/ironic/html/images/ironic-python-agent.kernel"
+FILESTOCACHE="/opt/dev-scripts/ironic/html/images/ironic-python-agent.initramfs /opt/dev-scripts/ironic/html/images/ironic-python-agent.kernel /opt/dev-scripts/ironic/html/images/ironic-python-agent.tar.headers /opt/dev-scripts/ironic/html/images/rhcos-ootpa-latest.qcow2 /opt/dev-scripts/ironic/html/images/rhcos-ootpa-latest.qcow2.headers /opt/dev-scripts/ironic/html/images/rhcos-ootpa-latest.qcow2.md5sum"
```

Comment:
We no longer consume the "latest" symlinks, only the versioned directory, so do we need to update this to reflect that?

Reply:
nvm, I retriggered CI and can see the right file does get cached:

```
# From /home/notstack/dev-scripts/logs/coreos-downloader.log
+ FILECACHED=rhcos-42.80.20190827.1-openstack.qcow2
+ '[' rhcos-42.80.20190827.1-openstack.qcow2 == rhcos-42.80.20190827.1-openstack.qcow2 ']'
+ exit 0
```